lua/darkrp_modules/levels/sv_addways.lua

```diff
@@ -1,62 +1,106 @@
-function PlayerDeath(victim, weapon, killer)
-	if(LevelSystemConfiguration.KillModule) then
-		if(victim != killer) then // Not a suicide
-			if(killer:IsPlayer()) then
-					local Lvictim = victim:Nick()
-					local Lmoney = DarkRP.formatMoney(LevelSystemConfiguration.TakeAwayMoneyAmount)
-					if(LevelSystemConfiguration.Friendly) then
-						if((killer:getDarkRPVar('level') or 1)<=(victim:getDarkRPVar('level') or 1)) then
-								local xpgot=killer:addXP(10*(victim:getDarkRPVar('level') or 1), true)
-								killer:addMoney(LevelSystemConfiguration.TakeAwayMoneyAmount)
-								DarkRP.notify(killer, 0,4,LevelSystemConfiguration.LangKillNotify)
-								if(victim:canAfford(LevelSystemConfiguration.TakeAwayMoneyAmount)) then
-									victim:addMoney(-LevelSystemConfiguration.TakeAwayMoneyAmount)
-									DarkRP.notify(victim, 0,4,LevelSystemConfiguration.LangDeath)					
-								end
-						else 
-							DarkRP.notify(killer,0,4,LevelSystemConfiguration.LangKillNotify2)
-						end
-					else
-						local xpgot = killer:addXP(10*(victim:getDarkRPVar('level') or 1),true)
+local function PlayerDeath(victim, weapon, killer)
+	if (LevelSystemConfiguration.KillModule) then
+		if (victim != killer) then -- Not a suicide
+			if (killer:IsPlayer()) then
+				local victime = victim:Nick()
+				local money = DarkRP.formatMoney(LevelSystemConfiguration.TakeAwayMoneyAmount)
+				local XP = killer:addXP( 10 * (victim:getDarkRPVar('level') or 1), true)
+
+				if (LevelSystemConfiguration.Friendly) then
+					if ((killer:getDarkRPVar('level') or 1) <= (victim:getDarkRPVar('level') or 1)) then
 						killer:addMoney(LevelSystemConfiguration.TakeAwayMoneyAmount)
-						DarkRP.notify(killer, 0,4,LevelSystemConfiguration.LangKillNotify3)
+						DarkRP.notify(killer, 0,4, string.format( LevelSystemConfiguration.LangKillNotify, XP, money, victime ))
+						if guthlogsystem then
+							guthlogsystem.addLog( "DarkRP Leveling System", "*"..killer.."* got &"..XP.."& XP and &"..money.."& for killing ?"..victime.."?" )
+						end
+
+						if (victim:canAfford(LevelSystemConfiguration.TakeAwayMoneyAmount)) then
+							victim:addMoney(-LevelSystemConfiguration.TakeAwayMoneyAmount)
+							DarkRP.notify(victim, 0,4, string.format( LevelSystemConfiguration.LangDeath, money ))
+						end
+					else 
+						DarkRP.notify(killer,0,4, string.format( LevelSystemConfiguration.LangKillNotify2, XP, victime ))
+						if guthlogsystem then
+							guthlogsystem.addLog( "DarkRP Leveling System", "*"..killer:Name().."* got &"..XP.."& XP for killing ?"..victime.."?" )
+						end
 					end
-						
+				else
+					killer:addMoney(LevelSystemConfiguration.TakeAwayMoneyAmount)
+					DarkRP.notify(killer, 0,4, string.format( LevelSystemConfiguration.LangKillNotify3, victime ))
```

Comment:
For example here you would be able to do DarkRP.getPhrase("LangKillNotify3", victim etc)